podaac/nose_tests_podaac/podaac_test.py

```diff
@@ -0,0 +1,83 @@
+
```

Comment:
Please add an Apache License v2.0 license header. You should be doing this for all of your source code files if you are coding at the Apache Software Foundation or related projects.